tmk_core/protocol/ps2_interrupt.c

```diff
@@ -61,12 +69,61 @@ static inline void    pbuf_enqueue(uint8_t data);
 static inline bool    pbuf_has_data(void);
 static inline void    pbuf_clear(void);
 
+#if defined(PROTOCOL_CHIBIOS)
+//------------------------------------------------
+// LEGACY EXT driver, for external interrupts
+// (with newer chibios we could use palLineEnableEvent and so forth...)
+// see: http://www.chibios.com/forum/viewtopic.php?t=3355#p25518 https://www.playembedded.org/blog/buttons-stm32/ 3.2.3 event/callback
+//
+void ps2_interrupt_service_routine(void);
+void extcb(EXTDriver *extp, expchannel_t channel) {
+    ps2_interrupt_service_routine();
+}
+
+/*
+  16 slots, one for each pin...
+  currently configured to listen on external changes on pad A8
+ */
+static EXTConfig extcfg = {
```

Comment:
This can be reduced to `EXTConfig extcfg = {0};`

Reply:
you're right, this is a remnant of development (was at one point != 0)